src/main/java/com/stripe/model/Product.java

```diff
@@ -29,6 +29,8 @@
 	SKUCollection skus;
 	Long updated;
 	String url;
+	String statementDescriptor;
```

Comment:
Aside from `id` and `object` which are always at the top of the list, we keep the attributes and their respective accessors sorted in alphabetical order.